hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/SixToFiveDowngradeHandler.java

```diff
@@ -79,6 +82,17 @@ public Map<ConfigProperty, String> downgrade(HoodieWriteConfig config, HoodieEng
     return updatedTableProps;
   }
 
+  private void rollbackPendingCommits(HoodieTableMetaClient metaClient, HoodieWriteConfig config,
+                                      HoodieEngineContext context, SupportsUpgradeDowngrade upgradeDowngradeHelper) {
+    Option<HoodieInstant> lastPendingInstantOpt = metaClient.getActiveTimeline().filterPendingExcludingCompaction().lastInstant();
```

Comment:
Instead of a while loop and reloading metaClient should I just executed rollback for all the instants returned by the API?